skimage/metrics/_structural_similarity.py

```diff
@@ -166,9 +166,14 @@ def structural_similarity(im1, im2,
 
     if np.any((np.asarray(im1.shape) - win_size) < 0):
         raise ValueError(
-            "win_size exceeds image extent.  If the input is a multichannel "
-            "(color) image, set channel_axis to the axis number corresponding "
-            "to the channels.")
+            "win_size exceeds image extent. ",
+            "Either ensure that your images are ",
+            "at least 7x7; or pass win_size explicitly ",
+            "in the function call, with an odd value ",
+            "less than or equal to the smaller side of your ",
+            "images. If your images are multichannel ",
+            "(with color channels), set channel_axis to ",
```

Comment:
small note. but: python does not need commas here.

Reply:
Very true, good catch! Waiting for your PR to fix this :smile:

Reply:
See the issue this closes as for why I won't be submitting that.

Reply:
Thanks @pomax for pointing out the issue. I fixed this in newly created PR here https://github.com/scikit-image/scikit-image/pull/5480 .